cli/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "analytics-node-cli",
+  "version": "0.0.1",
+  "description": "A command-line interface for Segment's analytics-node library",
+  "license": "MIT",
+  "repository": "segmentio/analytics-node",
+  "author": {
+    "name": "Segment",
+    "email": "tools+npm@segment.com",
+    "url": "https://segment.com"
+  },
+  "bin": {
+    "analytics": "./index.js"
+  },
+  "engines": {
+    "node": ">=4"
+  },
+  "scripts": {
+    "dependencies": "yarn",
+    "test": "standard",
+    "prepublish": "npm run check-deps",
+    "check-deps": "nsp check",
+    "np": "np --no-publish",
+    "release": "yarn run np"
+  },
+  "files": [
+    "index.js"
+  ],
+  "keywords": [
+    "analytics",
+    "cli",
+    "segment.io",
+    "segmentio",
+    "segment"
+  ],
+  "dependencies": {
+    "analytics-node": "^3.3.0"
```

Comment:
Can we make this depend the local version? Otherwise our end to end tests will be testing an outdated version